index.bs

```diff
@@ -1130,6 +1156,392 @@ The following extended attributes are applicable to interfaces:
+(matching <emu-nt><a href="#prod-MixinMembers">MixinMembers</a></emu-nt>),
+which are the [=constants=], [=regular operations=], [=regular attributes=],
+and an eventual [=stringifier=],
+that appear between the braces in the mixin declaration.
+
+These [=constants=], [=operations=], and [=attributes=]
+describe the behaviors that can be implemented by an object,
+as if they were specified on the [=interface=] that [=includes=] them.
+
+As with interfaces, the IDL for mixins can be split into multiple parts by using
+<dfn export>partial mixin</dfn> definitions
+(matching <emu-t>partial</emu-t> <emu-nt><a href="#prod-Mixin">Mixin</a></emu-nt>).
+The [=identifier=] of a partial mixin definition must be the same as the identifier of a mixin definition.
+All of the [=mixin members|members=] that appear on each of the partial mixin definitions
+are considered to be members of the mixin itself.
+
+<pre class="webidl" class="syntax">
+    mixin <mark>SomeMixin</mark> {
+      /* mixin_members... */
+    };
+
+    partial mixin <mark>SomeMixin</mark> {
+      /* mixin_members... */
+    };
+</pre>
+
+The order that members appear in has significance for property enumeration
+in the <a href="#es-namespaces">ECMAScript binding</a>.
+
+Note that unlike [=interfaces=] or [=dictionaries=], [=mixins=] do not create types.
+
+Of the extended attributes defined in this specification,
+only the [{{Exposed}}] and [{{SecureContext}}] extended attributes are applicable to mixins.
+
+An <dfn>includes statement</dfn> is a definition
+(matching <emu-nt><a href="#prod-IncludesStatement">IncludesStatement</a></emu-nt>)
+used to declare that all objects implementing an [=interface=] |I| (identified by the first [=identifier=])
+must additionally implement [=mixin=] |M| (identified by the second identifier).
+Interface |I| is said to <dfn>include</dfn> mixin |M|.
+
+<pre class="webidl" class="syntax">
+    interface_identifier includes mixin_indentifier;
+</pre>
+
+The first [=identifier=] must reference a [=callback interface|non-callback=] [=interface=] |I|.
+The second identifier must reference a [=mixin=] |M|.
+
+Each [=interface=] |I| that [=includes=] a [=mixin=] |M| must receive
+a copy of each of the [=mixin members|members=] of |M|.
+Each [=mixin member=] cooy is treated as if it had been declared on |I|.
+Its <dfn>host interface</dfn> is |I|.
+
+Note: In ECMAScript, this implies that each [=regular operation=]
+declared as a [=mixin member|member=] of [=mixin=] |M|,
+and exposed as a data property with a [=built-in function object=] value,
+is a distinct [=built-in function object=]
+in each [=interface prototype object=]
+whose associated [=interface=] [=includes=] |M|.
+Similarly, for [=attributes=], each copy of the accessor property has
+distinct [=built-in function objects=] for its getters and setters.
+
+The order of appearance of an [=includes statement=] does not matter.
```

Comment:
Not sure exactly what this means, but at least one interpretation is false, since A includes B; A includes C; is different from A includes C; A includes B; in terms of property enumeration order. (Kinda. It's all a bit fuzzy especially cross-specification.)

Reply:
Well, as you note, mixin order is undefined if you have multiple documents. So how do we handle this?

Reply:
We certainly shouldn't say it doesn't matter. Linking to an open issue for cross-document makes sense, while stating that it does matter within documents, perhaps.

Reply:
What do we do with the following pre-existing sentence though (which I must have been *heavily* inspired by):

> The order of appearance of an interface definition and any of its partial interface definitions does not matter.

Reply:
Yeah, it's known wrong. https://github.com/heycam/webidl/issues/432